bin/create-release.sh

```diff
@@ -2,6 +2,11 @@
 set -o errexit
 set -o nounset
 
+# prevent cd from printing the directory it changes to. This breaks
+# cd/pwd constructions (See **).
+unset CDPATH
+
+# (**)
 ROOT="$( cd "$( dirname "${BASH_SOURCE[0]}" )/../" && pwd )"
```

Comment:
Simplify all of this to:

```
# Make sure `cd` is not printing the directory it changes to.
ROOT="$( unset CDPATH; cd "$( dirname "${BASH_SOURCE[0]}" )/../" && pwd )"
```

And update the corresponding code in https://github.com/SUSE/scf/blob/develop/bin/create-release.sh#L5-L10 to match.

Reply:
Changed in ecaa4b61e3220c4f44a077437f130bb2ca77d733 — I'll do the SCF side once that's closer to being merged.